greek/use_the_rope/grab_the_ax/grab_the_ax

```diff
@@ -0,0 +1,4 @@
+Αρπάζεις το τσεκούρι το οποίο βρίσκεται κάτω απο το παράθυρο σε θήκη για περίπτωση φωτιάς και κόβεις το σχοινί.
+Με το που το κόβεις σου έρχεται στο μυαλό η εικόνα του παππού σου στο χωριό να κόβει τα ξύλα με το τσεκούρι αρχίζει να σε διακατέχει μια μανία και αρχίζεις να πετσοκόβεις τα ζαχαρωτά.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.